app/src/components/inputs/inputUserSearch/inputUserSearch.jsx

```diff
@@ -65,6 +65,7 @@ export const InputUserSearch = ({
   error,
   touched,
   placeholder,
+  ssoUsersOnly,
```

Comment:
The `ssoUsersOnly` is too specific for such a common component.
I suggest to use more common prop name here just like `creatable`.

Reply:
Good point! Instead of using ssoUsersOnly throughout the code, I'm now just using the creatable prop that was already in InputUserSearch - this makes more sense since it's a generic component.